src/utils.ts

```diff
@@ -770,17 +802,15 @@ export async function getEnvTemplateFilesAndAskOverwrite(
     if (error instanceof CancelOperationError) {
       telemetryContext.properties.result = 'Cancelled';
       telemetryContext.properties.errorMessage = error.message;
-      return;
-    } else {
-      throw error;
     }
+    throw error;
```

Comment:
This change looks differ from previous one. Do we need continue to throw CancelOperationError out?

Reply:
Yes. I handle CancelOperationError in the caller method ProjectEnvironmetConfiger.configureProjectEnvironment()

---
In reply to: [334725057](https://github.com/microsoft/vscode-iot-workbench/pull/775#discussion_r334725057) [](ancestors = 334725057)